### PR TITLE
Run tests on all versions + Docker for local testing

### PR DESCRIPTION
**Pull request by satanTime**
if you want to run tests locally - simply run `docker-compose up` container has node 10 and chrome browser. It's needed if your local node version is 12+, because some libs can't be compiled successfully on it.

### CODE REVIEW

**Comment by satanTime** [comment]
Wait, there's issue with puppeteer

**Comment by satanTime** [comment]
fixed, unfortunately, it has to be a dev dependency to avoid redownloading during all versions run.

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ before_script:
     - "sudo chmod 4755 /opt/google/chrome/chrome-sandbox"
 before_install:
   - npm cache clean --force
-script: npm run lint && npm run test
+script: npm run test:angular-versions
```
Comment (ike18t) [inline .travis.yml]: This all looks great! Do you mind putting the lint verification back in place? It is nice to have for the PR check.

Comment (satanTime) [reply .travis.yml]: it's here, but deep inside. for every version it executes `npm run build:all` which is `npm run lint && npm run test && npm run build`

Comment (satanTime) [reply .travis.yml]: and thanks for the invitation!

Comment (ike18t) [reply .travis.yml]: Thanks for all of your help!

**Pull request merged**